package.json

```diff
@@ -109,6 +110,8 @@
 		"start-build": "BROWSERSLIST_ENV=evergreen node build/server.js | bunyan -o short",
 		"start-jetpack-cloud": "CALYPSO_ENV=jetpack-cloud-development yarn run start",
 		"start-jetpack-cloud-p": "PORT=3001 CALYPSO_ENV=jetpack-cloud-development yarn run build-server && PORT=3001 CALYPSO_ENV=jetpack-cloud-development yarn run start-build",
+		"start-a8c-for-agencies": "CALYPSO_ENV=a8c-for-agencies-development yarn run start",
+		"start-a8c-for-agencies-p": "PORT=3001 CALYPSO_ENV=a8c-for-agencies-development yarn run build-server && PORT=3001 CALYPSO_ENV=a8c-for-agencies-development yarn run start-build",
```

Comment:
Should we use port 3002 here so it doesn't overlap with the Jetpack cloud, and we have the option of running both environments simultaneously?

Reply:
We use 3000 by default for Jetpack Cloud development. I initially thought we could use 3001 if 3000 is being used. But I think it makes sense to keep it 3002. I have updated it here: https://github.com/Automattic/wp-calypso/pull/87292/commits/dfbd6420c9d6822c84254a717a1d1b0c7b3fb0a5

@tyxla: I hope this is ok.

Reply:
We never really had a compelling reason to run multiple envs at the same time on the same machine. There could also be unforeseen conflicts because we're sharing the same build/asset directories and there are multiple processes watching for the same files/directories. I'd keep that as-is and change it only if we really need to.

Reply:
Also this is something I'd explore in a separate PR and try to make it benefit both Calypso Green and Calypso White.